Box.V2/Managers/BoxEventsManager.cs

```diff
@@ -39,7 +39,9 @@ public BoxEventsManager(IBoxConfig config, IBoxService service, IBoxConverter co
                                                                         DateTime? createdAfter = null,
                                                                         DateTime? createdBefore = null)
         {
-            var createdAfterString = createdAfter.HasValue ? createdAfter.Value.ToString(Constants.RFC3339DateFormat) : null;
+            var createdAfterString = createdAfter.Value.ToString(createdAfter.Value.Kind == DateTimeKind.Utc 
```

Comment:
Where are the integration test? What about the `o` format?

Reply:
More importantly, what about the proper conversion to UTC ? https://msdn.microsoft.com/en-us/library/system.datetime.touniversaltime(v=vs.110).aspx